capa/features/extractors/vmray/__init__.py

```diff
@@ -160,7 +170,12 @@ def _compute_monitor_processes(self):
                 self.sv2.processes[process.ref_parent_process.path[1]].os_pid if process.ref_parent_process else 0
             )
             self.monitor_processes[process.monitor_id] = VMRayMonitorProcess(
-                process.os_pid, ppid, process.monitor_id, process.image_name
+                process.os_pid,
+                ppid,
+                process.monitor_id,
+                process.image_name,
+                sanitize_string(process.filename),
+                sanitize_string(process.cmd_line),
```

Comment:
Is there a reason not to handle this via the `SanitizedString` as is done with `MonitorProcess`?

Reply:
`VMRayMonitorProcess` is this file's local dataclass (with no validators)

Reply:
Apologies, I should have been more specific. I meant add the `SanitizedString` validator to `Process`:
https://github.com/mandiant/capa/blob/7a8551bef997b1f309a052559b29e8e9556eea57/capa/features/extractors/vmray/models.py#L311-L321
Like you've done for `MonitorProcess`:
https://github.com/mandiant/capa/blob/7a8551bef997b1f309a052559b29e8e9556eea57/capa/features/extractors/vmray/models.py#L148-L164

Reply:
ah, of course, good catch, see updated PR